src/daemon/rpmostreed-transaction-types.c

```diff
@@ -626,12 +626,17 @@ import_local_rpm (OstreeRepo    *parent,
 
   /* let's just use the current sepolicy -- we'll just relabel it if the new
    * base turns out to have a different one */
-  {
-    glnx_unref_object GFile *root = g_file_new_for_path ("/");
-    policy = ostree_sepolicy_new (root, cancellable, error);
-    if (policy == NULL)
-      return FALSE;
-  }
+#if OSTREE_CHECK_VERSION(2017,4)
+  glnx_fd_close int rootfs_dfd = -1;
+  if (!glnx_opendirat (AT_FDCWD, "/", TRUE, &rootfs_dfd, error))
+    return FALSE;
+  policy = ostree_sepolicy_new_at (rootfs_dfd, cancellable, error);
```

Comment:
This is more of a general comment about this approach -- one issue here is that `#ifdef`ing on an unreleased version means that these paths can get shipped without ever actually being tested, right? We do build ostree from git for the `check` suite (so at least syntax errors are caught), but the other suites (which actually would test this path) are all going to be exercising the old path.

Anyway, just mentioning it. Obviously in this instance, it seems pretty low risk! Though if we do more of this, we might want to broaden our test coverage accordingly.